dffml/service/dev.py

```diff
@@ -1056,6 +1061,118 @@ class Bump(CMD):
     packages = BumpPackages
 
 
+class SphinxBuildError(Exception):
+    """
+    Raised when sphinx-build command exits with a non-zero error code.
+    """
+
+    def __str__(self):
+        return "[ERROR] Failed run sphinx, is it installed (pip install -U .[dev])?"
+
+
+@configdataclass
+class MakeDocsConfig:
+    port: int = field("PORT for the local docs server", default=8080)
+    http: str = field(
+        "If set  a SimpleHTTP server would be started to show generated docs.",
+        default=None,
+    )
+
+
+class MakeDocs(CMD):
+    """
+    Generate docs of the complete package 
+    """
+
+    CONFIG = MakeDocsConfig
+
+    async def _exec_cmd(self, cmd: List[str]) -> int:
+        print(f"$ {' '.join(cmd)}")
+        proc = await asyncio.create_subprocess_exec(*cmd)
+        await proc.wait()
+        return proc.returncode
+
+    async def run(self):
+        root = Path(__file__).parents[2]
+        pages_path = root / "pages"
```

Comment:
Take this as an argument via the config structure, then you can set it to a temporary directory for the tests

Reply:
Done